pyo3-build-config/src/util.rs

```diff
@@ -0,0 +1,59 @@
+//! Additional functionality enabled by crate features
```

Comment:
Should we have a "miscellaneous" module before having more than one optional function? I think a module that is specific to the python3-dll-a feature would nicer.